spec/system/acceptance/visit_home_controller_acceptance_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
```

Comment:
These tests don't have to be written as system specs or using rails helper as it's only using them for the capybara support. If we introduce an `acceptance_helper.rb` then we can add the capybara setup we need there and remove the rails dependency.